app/javascript/components/student/Editor.tsx

```diff
@@ -217,11 +210,13 @@ export function Editor({
 
       dispatch({ type: ActionType.CREATING_ITERATION })
 
-      fetchJSON(submission.links.submit, { method: 'POST' }).then(
-        (json: any) => {
-          dispatch({ type: ActionType.ITERATION_CREATED })
-        }
-      )
+      fetchJSON(submission.links.submit, {
+        method: 'POST',
+        body: JSON.stringify({}),
+      }).then((json: any) => {
+        const iteration = typecheck<Iteration>(json, 'iteration')
+        location.replace(iteration.links.self)
```

Comment:
Thank you for using links ❤️ .

1 question: is it intended to break the back button? If no, you'd probably want to use `pushState`.

Reply:
Added as an issue here: https://github.com/exercism/v3-website/issues/238